packages/components-react/heading-react/src/heading.ts

```diff
@@ -2,10 +2,14 @@ import { clsx } from 'clsx';
 import type { HTMLAttributes } from 'react';
 import { forwardRef, createElement } from 'react';
 
-export type HeadingAppearance = 'level-1' | 'level-2' | 'level-3' | 'level-4' | 'level-5' | 'level-6';
+export const headingLevels = [1, 2, 3, 4, 5, 6] as const;
+
+export type HeadingLevel = (typeof headingLevels)[number];
+
+export type HeadingAppearance = `level-${HeadingLevel}`;
 
 export interface HeadingProps extends HTMLAttributes<HTMLHeadingElement> {
-  level: 1 | 2 | 3 | 4 | 5 | 6;
+  level: HeadingLevel;
```

Comment:
In een volgende PR wil ik graag:

```suggestion
  level: HeadingLevel | number;
```

voor `level={parentLevel + 1}`

en ook die support voor `<p>`. Dit heb ik al eerder met Rian afgesproken zo.

Reply:
Ik heb een [issue](#102) aangemaakt.